section6Arrays/reverseString/main.go

```diff
@@ -0,0 +1,10 @@
+package reverseString
+
+func revStr(str string) string {
+	slc := []rune(str)
+	for i, j := 0, len(slc)-1; i < j; i, j = i+1, j-1 {
```

Comment:
I liked that you used two variables in the loop

Reply:
Ok

Reply:
Will fix from now on